ACMILabs/lens-reader
# #34 Update environment variable from museumos-prod to xos.

*Resolves issue #34*

Rename references from museumos-prod to xos.

### Acceptance Criteria

### Relevant design files
* None

### Testing instructions
1. Once XOS has been updated, restart `simons-desk` tap reader and see that taps are successfully sent to XOS: https://dashboard.balena-cloud.com/apps/1509309/devices

### DoD
For requester to complete:
